yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/ShareSessionReceipt.java

```diff
@@ -0,0 +1,206 @@
+        private Content otherPartyContent;
+        private String wrappedItemKeyId;
+        private byte[] rememberMeId;
+        private byte[] parentRememberMeId;
+        private byte[] wrappedKey;
+
+        private Builder() { }
+
+        @JsonProperty(Property.ID)
+        public Builder id(String id) {
+            this.id = id;
+            return this;
+        }
+
+        @JsonProperty(Property.SESSION_ID)
+        public Builder sessionId(String sessionId) {
+            this.sessionId = sessionId;
+            return this;
+        }
+
+        @JsonProperty(Property.TIMESTAMP)
+        public Builder timestamp(String timestamp) {
+            this.timestamp = timestamp;
+            return this;
+        }
+
+        @JsonProperty(Property.CONTENT)
+        public Builder content(Content content) {
+            this.content = content;
+            return this;
+        }
+
+        @JsonProperty(Property.OTHER_PARTY_CONTENT)
+        public Builder otherPartyContent(Content otherPartyContent) {
+            this.otherPartyContent = otherPartyContent;
+            return this;
+        }
+
+        @JsonProperty(Property.WRAPPED_ITEM_KEY_ID)
+        public Builder wrappedItemKeyId(String wrappedItemKeyId) {
+            this.wrappedItemKeyId = wrappedItemKeyId;
+            return this;
+        }
+
+        @JsonProperty(Property.REMEMBER_ME_ID)
+        public Builder rememberMeId(String rememberMeId) {
+            this.rememberMeId = decode(rememberMeId);
+            return this;
+        }
+
+        @JsonProperty(Property.PARENT_REMEMBER_ME_ID)
+        public Builder parentRememberMeId(String parentRememberMeId) {
+            this.parentRememberMeId = decode(parentRememberMeId);
+            return this;
+        }
+
+        @JsonProperty(Property.WRAPPED_KEY)
+        public Builder wrappedKey(String wrappedKey) {
+            this.wrappedKey = decode(wrappedKey);
+            return this;
+        }
+
+        public ShareSessionReceipt build() {
+            return new ShareSessionReceipt(this);
+        }
+
+        private static byte[] decode(String value) {
+            return Base64.getDecoder().decode(value);
+        }
+
+    }
+
+    private static class Property {
+
+        private static final String ID = "id";
+        private static final String SESSION_ID = "sessionId";
+        private static final String TIMESTAMP = "timestamp";
+        private static final String REMEMBER_ME_ID = "rememberMeId";
+        private static final String PARENT_REMEMBER_ME_ID = "parentRememberMeId";
+        private static final String CONTENT = "content";
+        private static final String OTHER_PARTY_CONTENT = "otherPartyContent";
+        private static final String WRAPPED_ITEM_KEY_ID = "wrappedItemKeyId";
+        private static final String WRAPPED_KEY = "wrappedKey";
+
+    }
+
+    private static class Content {
+
+        private byte[] profile;
+        private byte[] extraData;
```

Comment:
Same applies to these ones. they are "optional" and might not be present in the response.